relayer/chains/cosmos/query.go

```diff
@@ -316,42 +315,73 @@ func (cc *CosmosProvider) queryConsumerUnbondingPeriod(ctx context.Context) (tim
 
 	params := proposal.QueryParamsRequest{Subspace: "ccvconsumer", Key: "UnbondingPeriod"}
 
-	resICS, err := queryClient.Params(ctx, &params)
+	res, err := queryClient.Params(ctx, &params)
 
 	if err != nil {
 		return 0, fmt.Errorf("failed to make ccvconsumer params request: %w", err)
 	}
 
-	if resICS.Param.Value == "" {
+	if res.Param.Value == "" {
 		return 0, fmt.Errorf("ccvconsumer unbonding period is empty")
 	}
 
-	unbondingPeriod, err := strconv.ParseUint(strings.ReplaceAll(resICS.Param.Value, `"`, ""), 10, 64)
+	unbondingPeriod, err := strconv.ParseUint(strings.ReplaceAll(res.Param.Value, `"`, ""), 10, 64)
 	if err != nil {
 		return 0, fmt.Errorf("failed to parse unbonding period from ccvconsumer param: %w", err)
 	}
 
 	return time.Duration(unbondingPeriod), nil
 }
 
+func (cc *CosmosProvider) queryStakingUnbondingPeriod(ctx context.Context) (time.Duration, error) {
```

Comment:
this can be consolidated with `queryConsumerUnbondingPeriod`, maybe something like `queryParamsSubspaceTime(subspace string, key string) (time.Duration, error)`